src/helpers/shareHelper.js

```diff
@@ -38,26 +39,43 @@ export const openShare = async ({ message, title, url }) => {
 };
 
 export const shareMessage = (data, query) => {
+  const urls = mergeWebUrls({
+    webUrls: data.webUrls,
+    contact: data.contact,
+    contacts: data.contacts
+  })?.map(({ url }) => `Link: ${url}`);
+
+  const urlSection = urls.length ? urls.join('\n') : '';
+  const spacer = urlSection ? '\n' : '';
+
   const buildMessage = (query) => {
+    let baseMessage = '';
```

Comment:
what about just `message`? the method `buildMessage` builds a `message`? what was your thought with `base`?

Reply:
yes, it can be. there's nothing extra I want to say with `base`

---
yes it makes sense. this way we don't need the default section in the switch

here's the commit I've applied the two changes to
https://github.com/smart-village-solutions/smart-village-app-app/commit/8306f3e91b6144eb3771fd40c8b84cdde753ad68

Reply:
but empty line still missing :(

<img width="242" alt="image" src="https://github.com/smart-village-solutions/smart-village-app-app/assets/1942953/ef18d3c4-dc99-4f0e-b198-7fe36889f074">

Reply:
yes, you are right. i fixed it right away. 👍

https://github.com/smart-village-solutions/smart-village-app-app/commit/229db627ae2309d82340bb5c902e21cf30d1fa90